packages/server-core/src/projects/project-check-source-destination-match/project-check-source-destination-match.test.ts

```diff
@@ -80,12 +83,15 @@ describe('project-check-source-destination-match.test', () => {
 
     testUserApiKey = await app.service(userApiKeyPath).create({ userId: testUser.id })
 
-    await app.service(identityProviderPath).create(
+    await app.service(identityProviderPath)._create(
```

Comment:
You can use `identityProviderDataResolver.resolve()` method here to populate default values.